### DISCUSSION TITLE
Trying to create a new DB from sql statements

### DISCUSSION THREAD

**Discussion by tester0077** [mariuz/flamerobin#192] [General]
Using FR 0.9.3 (git hash 78287cd4) 64-bit version under Win 10 64-bit
I have an existing working FB database.
Saved the DDL script of the db and am now trying to create a new db from this script.
Have successfully created a new db using FR, using the usual user name, password, char set UTF-8.
After connecting to this new db, I try to execute the same sql statements I had saved from the existing db.
Executing the script seems to complete OK, but when I try to commit the transaction, I get an error - unfortunately FR does not seems to permit me to copy any text from the sql editor (sure would be nice if FR allowed me to paste & copy from this window) 
_IBPP::SQLexception
Context: transaction commit
Sql message: -607
This operation is not defined for system tables
Engine code 335544351_

As I am very new to FR, I am likely making a lot of (most likely incorrect) assumptions.
Can any one please help me sort this out?

**Comment by arvanus** [comment]
Hello @tester0077
Strange that you can't copy de text from the SQL Editor, try to right+click and Copy
If not, can you provide a example to test here?
Thanks
